### DISCUSSION TITLE
Lazer cannot import the skin just export from itself

### DISCUSSION THREAD

**Discussion by cdwcgt** [ppy/osu#26454] [Q&A] [answered]
![image](https://github.com/ppy/osu/assets/34775378/5cd48d14-ddfb-40e3-98da-ee2871b9e763)
like this.

https://files.catbox.moe/dbqg7r.osk
this the skin file which lazer export.

**Comment by bdach** [comment, marked as answer]
![1704804639](https://github.com/ppy/osu/assets/20418176/c44502bf-27c3-4472-b52c-4789ac17de7a)

I'm not sure how this is possible but the archive is corrupt. It has two entries for seemingly one directory. I can't extract this using a plain old archive manager even.

Until proven otherwise I'd say the most likely explanation is that you broke your db somehow when changing code. It's not like you bothered to provide log excerpts or reproduction scenario anyways.
